addons/wowl/static/src/core/localization.ts

```diff
@@ -0,0 +1,97 @@
+import { OdooBrowser } from "../env";
+import { Odoo } from "../types";
+
+export interface LocalizationParameter {
+  code: string;
+  date_format: string;
+  decimal_point: string;
+  direction: string;
+  grouping: any[];
+  multiLang: boolean;
+  thousands_sep: string;
+  time_format: string;
+}
+
+interface TranslatedTerms {
+  [key: string]: string;
+}
+
+/**
+ * Default values for localization parameters
+ */
+export const localizationParameter: LocalizationParameter = {
+  code: "en_US",
+  date_format: "%m/%d/%Y",
+  decimal_point: ".",
+  direction: "ltr",
+  grouping: [],
+  multiLang: false,
+  thousands_sep: ",",
+  time_format: "%H:%M:%S",
+};
+
+const translatedTerms: TranslatedTerms = {};
+
+export async function fetchLocalizationParameters(browser: OdooBrowser, odoo: Odoo): Promise<void> {
```

Comment:
it would be more elegant if it would return an object with the translated terms and the localization info, instead of modifying it in place

Reply:
how would you pass the localization partner to the user service?

Reply:
How would you do for _lt?